hydromt/models/model.py

```diff
@@ -246,32 +230,35 @@ def build(
                 }
 
         """
-        opt = opt or {}
-        opt = self._check_get_opt(opt)
-
-        # merge cli region and res arguments with opt
-        if region is not None:
-            if self._CLI_ARGS["region"] not in opt:
-                opt = {self._CLI_ARGS["region"]: {}, **opt}
-            opt[self._CLI_ARGS["region"]].update(region=region)
```

Comment:
This functionality is not removed, we have written it in a different way. In this PR `region` can still come from the command line. But it will always be passed to `region.create`, and not to `setup_basemaps` if this was a plugin model.